org.flightgear.FlightGear.appdata.xml

```diff
@@ -42,7 +36,6 @@ SentUpstream: 2014-09-17
     <p>
       Más populares hardware de control de vuelo, como yokes y sticks son compatibles. Varios monitores de configuración y multijugador también se destaca.
     </p>
-    </description>
```

Comment:
This shouldn't be remove, nur the other description tag should get a closing tag as well.